### ISSUE TITLE
Unable to get demo app to work with RTSP stream

### ISSUE THREAD

**Issue by jpage4500** [alexeyvasilyev/rtsp-client-android#1]
Hi @alexeyvasilyev 

First off - thanks for writing this library! I'm hoping it can solve the problem I've found which is VideoView doesn't support RTSP streams that require authentication.

However, the demo isn't working for me. I have an Amcrest AD110 [video doorbell](https://www.amazon.com/Amcrest-Doorbell-Weatherproof-Wide-Angle-AD110/dp/B07ZJS3L5Y) which supports an RTSP stream.

The format of the RTSP URL is: `rtsp://user:password@192.168.0.208:554/cam/realmonitor?channel=1&subtype=1`

I've verified this stream works in VLC. For the demo app, I removed the `user:password` part and entered them separately.

```
rtsp://192.168.0.208:554/cam/realmonitor?channel=1&subtype=1
user
password
```

But, when hitting the Start button I get this in the logs:

```
 Starting RTSP thread...
 Connecting to 192.168.0.208:554...
 onRtspClientStarted()
 java.net.SocketException: socket failed: EACCES (Permission denied)
 	at java.net.Socket.createImpl(Socket.java:492)
 	at java.net.Socket.connect(Socket.java:619)
 	at com.alexvas.utils.NetUtils.createSocketAndConnect(NetUtils.java:98)
 	at com.alexvas.rtsp.demo.ui.live.LiveFragment$RtspThread.run(LiveFragment.kt:171)
 onRtspClientStopped()

```

Any ideas?

**Comment by alexeyvasilyev** [comment]
Fixed in 1.2.7.

**Comment by jpage4500** [comment]
worked perfectly! Thanks
